openwisp_controller/config/base/device_group.py

```diff
@@ -0,0 +1,40 @@
+import collections
+
+import jsonschema
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from jsonfield import JSONField
+from jsonschema.exceptions import ValidationError as SchemaError
+
+from openwisp_users.mixins import OrgMixin
+from openwisp_utils.base import TimeStampedEditableModel
+
+from .. import settings as app_settings
+
+
+class AbstractDeviceGroup(OrgMixin, TimeStampedEditableModel):
+    name = models.CharField(max_length=60, unique=True, null=False, blank=False)
+    description = models.TextField(blank=True, help_text=_('internal notes'))
+    context = context = JSONField(
```

Comment:
why context and not `meta_data`?

Reply:
I was facing this error while creating migrations, so went ahead with quick and dirty way for first iteration. 

```
SystemCheckError: System check identified some issues:

ERRORS:
config.DeviceGroup.meta_data: (models.E006) The field 'meta_data' clashes with the field 'meta_data' from model 'config.devicegroup'.
```

i still need to look into why this is happening

Reply:
It was a really silly thing. Fixed it!